doc/developer-guides/hld/hld-overview.rst

```diff
@@ -351,7 +354,7 @@ the standard Linux char device API (ioctl) to access HSM
 functionalities. HSM communicates with the ACRN hypervisor through
 hypercall or upcall interrupts.
 
-Refer to the HSM chapter for more details.
+Refer to the HSM chapter for more details. <link?>
 
```

Comment:
Is there an HSM chapter?

Reply:
Here's a link to the section (https://projectacrn.github.io/latest/developer-guides/hld/hld-devicemodel.html#hsm) but you'll need to add a label in hld-devicemodel.rst at that point so you can add a direct link to it here.